src/stabilization_guide.md

```diff
@@ -183,6 +183,73 @@ if self.tcx.sess.features.borrow().pub_restricted && something { /* XXX */ }
 if something { /* XXX */ }
 ```
 
+## Stabilization PR for Library Features
+
+Once we have decided to stabilize a feature, we need to have
+a PR that actually makes that stabilization happen. These kinds
+of PRs are a great way to get involved in Rust, as they're typically
+small -- just updating attributes.
+
+Here is a general guide to how to stabilize a feature --
+every feature is different, of course, so some features may
+require steps beyond what this guide talks about.
+
+### Update the stability attributes on the items
+
+Library items are marked unstable via the `#[unstable]` attribute, like this:
+
+```rust,ignore
+#[unstable(feature = "total_cmp", issue = "72599")]
+pub fn total_cmp(&self, other: &Self) -> crate::cmp::Ordering { ... }
+```
+
+You'll need to change that to a `#[stable]` attribute with a version:
+
+```rust,ignore
+#[stable(feature = "total_cmp", since = "1.61.0")]
+```
+
+Note that, the version number is updated to be the version number
+of the stable release where this feature will appear. This can be
+found by consulting [the forge](https://forge.rust-lang.org/#current-release-versions).
+Specifically, you'll want to use the version labelled "Nightly".
```

Comment:
Although this is a perfectly fine way to check the version, it might be more reliable to consult [`src/version`](https://github.com/rust-lang/rust/blob/master/src/version) on master.  That can help avoid confusion with the version bump that happens on the Friday before release.

I might also suggest *predicting* the version that it will land in.  If it looks like it might land around the Friday before release, then there is risk that it will miss the release window.

Finally, I might suggest putting this discussion of determining the version into a dedicated section and removing the duplicate discussion above with lang stabilizations. For the most part, the process is the same between them. (And the lang one is not written as clearly.)

It is not uncommon that PRs are approved with the wrong version in them, and then there is confusion if it isn't caught, or triggers busywork for backports and such.  It would be nice to be a little more careful about that.